jiant/trainer.py

```diff
@@ -862,16 +864,19 @@ def _calculate_validation_performance(
                         )
                         inputs = batch["inputs"]["words"][0][1:]
                         gold = batch["targs"]["words"][0][1:]
-                        logits = out["logits"]
-                        output = logits.max(2)[1][0]
-                        input_string, gold_string, output_string = task.get_prediction(
-                            voc_src, voc_trg, inputs, gold, output
-                        )
-                        log.info("\tInput:\t%s", input_string)
-                        log.info("\tGold:\t%s", gold_string)
-                        log.info("\tOutput:\t%s", output_string)
+                        # logits = out["logits"]
```

Comment:
delete

also, is `batch_num == 1` just so you don't print every batch ?

Reply:
Done. Also: yes, I just wanted one example to see what's going on.